src/index.ts

```diff
@@ -423,8 +432,29 @@ export class UnleashClient extends TinyEmitter {
         await this.storage.save(storeKey, toggles);
     }
 
+    private isUpToDate() {
+        if (this.togglesStorageTTL <= 0 || this.toggles.length === 0) {
```

Comment:
This will still hammer the API if the toggles length is actually is 0, which can happen in some cases:

- A fresh Unleash installation
- you are connecting it to a empty project

Reply:
Ok we can remove that, we thought we didn't want to set empty toggles in the cache but I see your point.